engine/dlib/src/dmsdk/dlib/log.h

```diff
@@ -211,6 +219,46 @@ void dmLogInternal(dmLogSeverity severity, const char* domain, const char* forma
 #define dmLogOnceFatal(format, args... ) dmLogOnceCritical(dmLogFatal, format, ## args )
 #endif
 
+/*# dmLogListener callback typedef
```

Comment:
@JCash maybe it make sense to have a separate namespace `dmLog` for

- dmLogSeverity enum  (dmLog::SEVERITY_DEBUG and so on...)
- dmLogListener  (dmLog::Listener)
- dmLogRegisterListener (dmLog::RegisterListener)
- dmLogUnregisterListener (dmLog::UnregisterListene)
- dmLogSetlevel (dmLog::Setlevel)

(but not for log functions)

What do you think?

Reply:
Yes, I think a new `dmLog::` namespace is a good idea.
We should also have the same functions there: dmLog::LogInfo() etc for consistency.
The "dmLogInfo()" etc can be defines that point to those functions

Reply:
I added `dmLog::` namespace. Thx